parent/pom.xml

```diff
@@ -0,0 +1,532 @@
+                <version>${ant.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.ant</groupId>
+                <artifactId>ant-launcher</artifactId>
+                <version>${ant.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.ant</groupId>
+                <artifactId>ant-commons-logging</artifactId>
+                <version>${ant.version}</version>
+                <exclusions>    <!-- we use jcl-over-slf4j -->
+                    <exclusion>
+                          <groupId>commons-logging</groupId>
+                          <artifactId>commons-logging-api</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>${junit.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-all</artifactId>
+                <version>1.10.19</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <licenses>
+        <license>
+            <name>LGPL, version 2.1</name>
+            <url>http://www.gnu.org/licenses/licenses.html</url>
+            <distribution>repo</distribution>
+        </license>
+        <license>
+            <name>ASL, version 2</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:https://github.com/java-native-access/jna</connection>
+        <developerConnection>scm:git:ssh://git@github.com/java-native-access/jna.git</developerConnection>
+        <url>https://github.com/java-native-access/jna</url>
+    </scm>
+
+    <developers>
+        <developer>
+            <id>twall</id>
+            <name>Timothy Wall</name>
+            <roles>
+                <role>Owner</role>
+            </roles>
+        </developer>
+    </developers>
+    
+    <build>
+        <defaultGoal>install</defaultGoal>
+
+        <pluginManagement>
+            <plugins>
+                <!--This plugin's configuration is used to store Eclipse 
+                    m2e settings only. It has no influence on the Maven build itself. -->
+                <plugin>
+                    <groupId>org.eclipse.m2e</groupId>
+                    <artifactId>lifecycle-mapping</artifactId>
+                    <version>1.0.0</version>
+                    <configuration>
+                        <lifecycleMappingMetadata>
+                            <pluginExecutions>
+                                <pluginExecution>
+                                    <pluginExecutionFilter>
+                                        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
I don't use Eclipse, so I can't test or argue this particular Eclipse specific usage. :)